pkg/image/prune/imagepruner.go

```diff
@@ -491,7 +491,15 @@ func (p *imagePruner) Run(pruneImage ImagePruneFunc, pruneStream ImageStreamPrun
 	pruneImages(p.g, imageNodes, pruneImage, pruneStream, pruneManifest)
 
 	layerNodes := layerNodeSubgraph(allNodes)
-	pruneLayers(p.g, layerNodes, pruneLayer, pruneBlob)
+	registryURL := ""
+	if len(imageNodes) > 0 {
+		if ref, err := imageapi.ParseDockerImageReference(imageNodes[0].Image.DockerImageReference); err == nil {
```

Comment:
Also, did we ever decided on whether we wanted an endpoint available on the API server that would give a location for hitting the internal docker registry?  Feels like that would be useful.